data/sql_setup/prepare_schedule_a.sql

```diff
@@ -3,84 +3,298 @@ drop table if exists ofec_sched_a_nightly_retries;
+                increased_limit = $1.increased_limit,
+                action_cd = $1.action_cd,
+                action_cd_desc = $1.action_cd_desc,
+                tran_id = $1.tran_id,
+                back_ref_tran_id = $1.back_ref_tran_id,
+                back_ref_sched_nm = $1.back_ref_sched_nm,
+                schedule_type = $1.schedule_type,
+                schedule_type_desc = $1.schedule_type_desc,
+                line_num = $1.line_num,
+                image_num = $1.image_num,
+                file_num = $1.file_num,
+                link_id = $1.link_id,
+                orig_sub_id = $1.orig_sub_id,
+                filing_form = $1.filing_form,
+                rpt_tp = $1.rpt_tp,
+                rpt_yr = $1.rpt_yr,
+                election_cycle = $1.election_cycle,
+                timestamp = CURRENT_TIMESTAMP,
+                pg_date = CURRENT_TIMESTAMP,
+                pdf_url = image_pdf_url($1.image_num),
+                contributor_name_text = to_tsvector(concat($1.contbr_nm, '' '', $1.contbr_id)),
+                contributor_employer_text = to_tsvector($1.contbr_employer),
+                contributor_occupation_text = to_tsvector($1.contbr_occupation),
+                is_individual = is_individual($1.contb_receipt_amt, $1.receipt_tp, $1.line_num, $1.memo_cd, $1.memo_text),
+                clean_contbr_id = clean_repeated($1.contbr_id, $1.cmte_id),
+                two_year_transaction_period = get_cycle($1.rpt_yr),
+                line_number_label = expand_line_number($1.filing_form, $1.line_num)
+            WHERE
+                %I.sub_id = $1.sub_id',
+                child_table_name, child_table_name) USING view_row;
+            PERFORM increment_aggregates(view_row);
         end if;
-
-        return new;
-    elsif tg_op = 'UPDATE' then
-        select into view_row * from fec_fitem_sched_a_vw where sub_id = new.sub_id;
-
-        if FOUND then
-            two_year_transaction_period = cast(get_cycle(view_row.rpt_yr) as smallint);
-
-            if two_year_transaction_period >= start_year then
-                delete from ofec_sched_a_queue_new where sub_id = view_row.sub_id;
-                insert into ofec_sched_a_queue_new values (view_row.*, timestamp, two_year_transaction_period);
-            end if;
-        end if;
-
-        return new;
     end if;
+
+    RETURN NULL; -- result is ignored since this is an AFTER trigger
 end
 $$ language plpgsql;
 
 
--- Create trigger to maintain Schedule A queues deletes and updates
+-- Create trigger to maintain Schedule A deletes and updates
 -- These happen before a row is removed/updated so that we can leverage pulling
 -- the new record information from the view itself, which contains the data in
 -- the structure that our tables expect it to be in.
-create or replace function ofec_sched_a_delete_update_queues() returns trigger as $$
+create or replace function ofec_sched_a_delete_update() returns trigger as $$
```

Comment:
Same note as before regarding the older version of the function - we shouldn't let it hang out.